lib/dependencies/SystemRuntimeModule.js

```diff
@@ -9,18 +9,6 @@ const RuntimeGlobals = require("../RuntimeGlobals");
 const RuntimeModule = require("../RuntimeModule");
 const Template = require("../Template");
 
-const SystemPolyfill = Template.asString([
-	`${RuntimeGlobals.system} = {`,
-	Template.indent([
-		"import: function () {",
-		Template.indent([
-			"throw new Error('System.import cannot be used indirectly');"
-		]),
-		"}"
-	]),
-	"}"
```

Comment:
Semicolon was missing. I also moved the block to match the style of the other runtime modules